Tests/UnitTests/Plugins/Json/VolumeBoundsJsonConverterTests.cpp

```diff
@@ -0,0 +1,140 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Definitions/Units.hpp"
+#include "Acts/Geometry/ConeVolumeBounds.hpp"
+#include "Acts/Geometry/CuboidVolumeBounds.hpp"
+#include "Acts/Geometry/CutoutCylinderVolumeBounds.hpp"
+#include "Acts/Geometry/CylinderVolumeBounds.hpp"
+#include "Acts/Geometry/TrapezoidVolumeBounds.hpp"
+#include "Acts/Plugins/Json/ActsJson.hpp"
+#include "Acts/Plugins/Json/VolumeBoundsJsonConverter.hpp"
+
+#include <fstream>
+#include <iostream>
+
+using namespace Acts;
+
+BOOST_AUTO_TEST_SUITE(VolumeBoundsJsonConverter)
+BOOST_AUTO_TEST_CASE(Cuboid) {
+  std::ofstream out;
```

Comment:
in principle you could open the file here already with an constructor arg `"CuboidVolumeBounds.json"`

same below